SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -72,8 +79,10 @@ struct ConfigureConnection
 	FTCHARToUTF8 ProtocolLogPrefix;
 	Worker_ComponentVtable DefaultVtable{};
 	Worker_CompressionParameters EnableCompressionParams{};
-	Worker_KcpTransportParameters UpstreamParams{};
-	Worker_KcpTransportParameters DownstreamParams{};
+
+#if WITH_EDITOR
+	Worker_HeartbeatParameters HeartbeatParams{ WORKER_DEFAULTS_HEARTBEAT_INTERVAL_MILLIS, MAX_int64 };
```

Comment:
Looks like it's gonna take quite a while to timeout with `MAX_int64`? Is it safe to do so?

Reply:
Yep, in this instance we never want to timeout.